docs/vscode.md

```diff
@@ -74,7 +74,7 @@ Add the following to (or create) the `.vscode/settings.json` file for VSCode to
     "search.useIgnoreFiles": false,
     "search.exclude": {
         "**/.git": true,
-        "baserom/**": true,
+        "baseroms/gc-eu-mq-dbg/segments/**": true,
```

Comment:
does this syntax exist/work:
```suggestion
        "baseroms/*/segments/**": true,
```

Reply:
should probably just exclude the entire baseroms folder? at least I cant think of anything I would want from there to show up in my searches